src/components/Buttons.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Buttons = () => (
+  <div className="btn-container">
+    <div className="result-output">0</div>
+    <button className="input-btn" type="submit">AC </button>
+    <button className="input-btn" type="submit">+/-</button>
+    <button className="input-btn" type="submit">%</button>
+    <button className="input-btn operations" type="submit">/</button>
```

Comment:
- [ ] Kindly use the correct division sign in the design: &div;. This will be important for future calculation feature